python/lsst/afw/table/slots.cc

```diff
@@ -24,26 +24,49 @@
 //#include <pybind11/operators.h>
 //#include <pybind11/stl.h>
 
+#include "lsst/afw/table/slots.h"
+
 namespace py = pybind11;
 
-namespace lsst {
-namespace afw {
-namespace table {
+using namespace lsst::afw::table;
+using namespace pybind11::literals;
+
+/**
+Declare standard methods for subclasses of SlotDefinition (but not SlotDefinition itself).
+*/
+template<typename SlotDefSubclass>
+void declareSlotDefinitions(py::class_<SlotDefSubclass, SlotDefinition> & pyClass) {
```

Comment:
Argument should be named `cls`. Same convention for class naming applies in declare functions.